docs/configuration.md

```diff
@@ -147,6 +147,7 @@ by using `_NO_` in the macro, e.g. `CATCH_CONFIG_NO_CPP17_UNCAUGHT_EXCEPTIONS`.
     CATCH_CONFIG_WINDOWS_SEH                // Enable SEH handling on Windows
     CATCH_CONFIG_FAST_COMPILE               // Sacrifices some (rather minor) features for compilation speed
     CATCH_CONFIG_POSIX_SIGNALS              // Enable handling POSIX signals
+    CATCH_CONFIG_GETENV                     // System has a working `getenv`
```

Comment:
I prefer appending to the list, unless some of the options form a logical group.

Also adding new option needs a version placeholder (see `contributing.md` for more).